README.md

```diff
@@ -246,12 +246,15 @@ If your installation didn't set up the helper processes required by CommCare HQ
 to automatically run on system startup, you need to run them manually:
 
     redis-server /path/to/redis.conf
+
     /path/to/unzipped/elasticsearch/bin/elasticsearch &
+
     /path/to/couchdb/bin/couchdb &
 
 Then run the following separately:
 
-    # MacOS Asynchronous task scheduler
+    # Setting up the asynchronous task scheduler
+    # For Mac / Linux
     ./manage.py celeryd --verbosity=2 --beat --statedb=celery.db --events
```

Comment:
I think this works for Linux too. When I ran it, many errors were thrown, but half the manage.py commands I ran threw errors I was then told to ignore by Cory